lineapy/utils/utils.py

```diff
@@ -55,8 +55,8 @@ def get_value_type(val: Any) -> Optional[ValueType]:
     if they are, then we can reference them.
 
     Note:
+
```

Comment:
Should move this to a comment `TODO`

Reply:
I'm actually going to revert this comment back to the original. 

The TODO is actually a TODO and is actionable and should be left as a TODO. 

The "Watch out for error here" is not actionable and should be a Note.